17.md

```diff
@@ -0,0 +1,34 @@
+NIP-17
+======
+
+Sensitive Content / Content Warning
+-----------------------------------
+
+`draft` `optional` `author:fernandolguevara`
+
+Like in mastodon, `content-warning` tag enables users to specify if the event's content needs to be approved by readers to be shown.
```

Comment:
**Like in mastodon,** should be removed, it doesn't add anything...
The `content-warning` ...